src/Integration.Vsix.UnitTests/CFamily/CFamilyIssueToAnalysisIssueConverterTests.cs

```diff
@@ -238,15 +288,26 @@ public void Convert_HasMessageParts_LineHashCalculatedForIssueAndLocations()
             firstLocation.LineHash.Should().Be(firstLocationHash);
         }
 
+        
         [TestMethod]
         public void Convert_HasMessageParts_LineHashCalculatedForNonFileLevelLocationsOnly()
         {
+            const string fileLevelLocationFilePath = "file1.cpp";
+            const string nonFileLevelLocationFilePath = "file2.cpp";
+            const int nonFileLevelLocationLine = 20;
+
+            var textSnapshot = CreateTextSnapShot(Array.Empty<ITextSnapshotLine>());
+            var textBuffer = CreateTextBuffer(textSnapshot);
+            var textDocument = CreateTextDocument(textBuffer);
+
             var fileSystemMock = CreateFileSystemMock();
+            fileSystemMock.Setup(x => x.File.Exists(fileLevelLocationFilePath)).Returns(true);
```

Comment:
For file level issues we need to get the snapshot so we can check if there are any leading empty lines.